Microsoft.Toolkit.Uwp.UI.Controls/EmojiIcon/EmojiIcon.xaml

```diff
@@ -0,0 +1,18 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls">
+
+    <Style TargetType="controls:EmojiIcon">
+        <Setter Property="EmojiSkinTone" Value="Default" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:EmojiIcon">
+                    <TextBlock x:Name="EmojiText"
```

Comment:
To follow suit with the SymbolIcon and provide the same functionality the TextBlock should be wrapped in a grid
```
<Grid>
    <TextBlock  FontFamily="Segoe UI Emoji" FontSize="20" FontStyle="Normal" FontWeight="Normal" HorizontalAlignment="Stretch" TextAlignment="Center" VerticalAlignment="Center"/>
</Grid>
```